### PR TITLE
Update WDLs to adhere with the miniwdl requirements

### PR DESCRIPTION
**Pull request by VJalili**
- [x] Remove some duplicate declaration of arguments;
- [x] Remove passing incorrect arguments (setting values to args that do not exist);
- [x] Change flatten style to adhere with miniwdl requirements;
- [x] Refactor `SVConcordance` task to `SVConcordanceTask` to avoid workflow and task having the same name.

All the wdls are tested with miniwdl and womtool. Womtool testing is run as the following:

```
$ echo '{ "google_project_id": "my-google-project-id", "terra_billing_project_id": "my-terra-billing-project" }' > inputs/values/google_cloud.my_project.json
$ scripts/inputs/build_default_inputs.sh -d . -c google_cloud.my_project
$ scripts/test/validate.sh -d . -j womtool-72.jar
```

### CODE REVIEW

File: wdl/GATKSVPipelineSingleSampleTest.wdl [hunk 1]
```diff
@@ -18,7 +18,12 @@ workflow GATKSVPipelineSingleSampleTest {
   call module.GATKSVPipelineSingleSample {
     input:
       sample_id = case_sample,
-      ref_samples = ref_samples
+
+      # ref_samples = ref_samples
+      # The above line is commented out to only
+      # to pass the miniwdl validation. The argument to
+      # pass here is ref_samples_list which is a file
+      # that contains a list of the samples.
```
Comment (mwalker174) [inline wdl/GATKSVPipelineSingleSampleTest.wdl:26]: You can go ahead and delete this WDL actually.

**Review by mwalker174 — APPROVED** [review]
Thanks @VJalili looks ok to me, just one request.

**Review by cwhelan — APPROVED** [review]
Looks good to me too.

File: wdl/GATKSVPipelineSingleSampleTest.wdl (hunk 1 above)
Comment (VJalili) [reply wdl/GATKSVPipelineSingleSampleTest.wdl]: That will make it simpler! Thanks.

**Comment by VJalili** [comment]
Thank you @mwalker174 and @cwhelan!

**Pull request merged**